src/resources/views/crud/fields/rate.blade.php

```diff
@@ -0,0 +1,67 @@
+<!-- radio -->
```

Comment:
Personally I think a better name for this would be ```star```, rather than ```rate```. Since ```rate``` can take many forms (ex: numbers 1-10, stars, thumbs, tomatoes?!), I think ```star``` is a lot more specific. I don't see anybody confusing a ```star``` field with anything else 😄

Reply:
Really happy with your response, in the same way I think you are probably right, maybe it could be more better if it was some kind of official package for fields, columns widgets and filter to give more trust to developers, you can think of it like a company expanding its sites. Like that we can focus more on developing the core package and maintaining more the small features.

Reply:
Exactly! I'm glad you understand. 

I think it's best that "extra" features are separate packages, that you install when you need it. And I think it's best for the add-ons to be as small as possible, so that they're easy to maintain and easy to install without causing conflicts: if one add-on had 10 fields, and another add-on had 10 fields... there's a change that both would have a field with the same name. It wouldn't break the admin panel, but it would be confusing.

If this actually happens, and we end up creating new add-ons for every "extra" field we think of, this would cause another inconvenience - that you have to install all the fields you need. But it's easily solvable with an official registry - we can ask people upon installation what "add-ons" they also want to install. That would have all the benefits - easy to install everything, but also easy to maintain.

So you're planning on creating the add-on? If so, feel free to reach out in this thread if you have any questions or want early feedback - happy to help.